File: getTitle/python/get_title.py
import requests # HTTP request library
import re       # regular expressions

def get_title(url):
  """Get the <title> from the HTML at the url"""
  r = requests.get(url)
  p = re.compile('<title>(.*?)</title>')
  s = p.search(r.text)
  return s.group(1)

File: getTitle/js/get-title.js
// get-title.js

// 3rd party library for HTTP requests
const rp = require('request-promise');

const getTitle = url => {
  return rp.request(url)
    .then(html => {
      const regex = /<title>(.*?)<\/title>/;
      let title = html.match(regex);
      return title[1];
    });
}

module.exports = getTitle;